app/screens/home/RouteList.tsx

```diff
@@ -440,15 +500,27 @@ const RouteList = ({
             </DeFiButton>
           </View>
           <SpacerV />
-          <View style={AppStyles.containerHorizontal}>
-            <DeFiButton mode="contained" onPress={() => setIsStakingRouteEdit(true)} style={{ flex: 1 }}>
-              {t("model.route.staking")}
-            </DeFiButton>
+          {session?.isBetaUser ? (
+            <View style={AppStyles.containerHorizontal}>
+              <DeFiButton mode="contained" onPress={() => setIsStakingRouteEdit(true)} style={{ flex: 1 }}>
+                {t("model.route.staking")}
+              </DeFiButton>
+              <SpacerH />
+              <DeFiButton mode="contained" onPress={() => setIsCryptoRouteEdit(true)} style={{ flex: 1 }}>
+                {t("model.route.crypto")}
+              </DeFiButton>
           </View>
+          ) : (
+            <View style={AppStyles.containerHorizontal}>
+              <DeFiButton mode="contained" onPress={() => setIsStakingRouteEdit(true)} style={{ flex: 1 }}>
+                {t("model.route.staking")}
+              </DeFiButton>
+            </View>
+          )}
```

Comment:
Is it required to duplicate the staking button? Or could you just put the `isBetaUser` around the Crypto button?

Reply:
good point. this is unnecessary

Reply:
resolve in #125